upup/pkg/fi/cloudup/awstasks/network_load_balancer.go

```diff
@@ -362,6 +362,12 @@ func (e *NetworkLoadBalancer) Find(c *fi.Context) (*NetworkLoadBalancer, error)
 				}
 				sm.PrivateIPv4Address = a.PrivateIPv4Address
 			}
+			if a.AllocationId != nil {
+				if sm.AllocationID != nil {
+					return nil, fmt.Errorf("NLB has more then one AllocationID per subnet, which is unexpected. This is a bug in kOps, please open a GitHub issue.")
```

Comment:
Nit, but nice to fix. Same for above.
```suggestion
					return nil, fmt.Errorf("NLB has more than one AllocationID per subnet, which is unexpected. This is a bug in kOps, please open a GitHub issue.")
```